docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,757 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Modules with prebuilt {anomaly-detect} jobs
```

Comment:
It's never been entirely clear to me what these "modules" were, so unless we can define it I think we should remove it:
```suggestion
== Prebuilt {anomaly-detect} jobs
```